### PR TITLE
An feed test

### PR DESCRIPTION
**Pull request by arunnandewal**
completed feed test cases please review

### CODE REVIEW

File: thunderbird/src/main/java/zeta/android/thunderbird/api/devapi/response/feed/FeedShareResponse.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class FeedShareResponse {
+
+    @SerializedName("og:title")
+    public String ogTitle;
+
+    @SerializedName("og:url")
+    public String ogUrl;
+
+    @SerializedName("og:image:url")
+    public String ogImageUrl;
```
Comment (manjunathc23) [inline thunderbird/src/main/java/zeta/android/thunderbird/api/devapi/response/feed/FeedShareResponse.java]: `public final` please

**Review by manjunathc23 — CHANGES_REQUESTED** [review]
@arunnandewal : You may want to consider to mark final for all API layer variables.

File: thunderbird/src/main/java/zeta/android/thunderbird/api/devapi/response/feed/FeedShareResponse.java (hunk 1 above)
Comment (muditpant13) [reply thunderbird/src/main/java/zeta/android/thunderbird/api/devapi/response/feed/FeedShareResponse.java]: +1

**Comment by manjunathc23** [comment]
This was rebased and merged into master. Thanks @arunnandewal

**Pull request closed without merging**